registry/data-models/models.py

```diff
@@ -8,54 +8,84 @@
 For each feature registry provider/implementation, they will extend this abstract
 data models and backend API.
 Diagram of the data models:
-                     +-----------------+
-                     |     Project     |
-                     +-----------------+
-                             |  
-                             |
-                             | 
-                +------------------------------+    
-                |                              |      
-                | 1:n                          | 1:n         
-       +------------------+           +------------------+
-       | FeatureInterface |           |     Anchor       |
-       +------------------+           +------------------+
-               ｜                             ｜        |
-               ｜ 1:n                         ｜1:n     |
-               +------------------------------+        |
-                             ｜                         |
-                             ｜1:n                      | contains
-                      +---------------+           +---------------+
-                      |    Feature    | --------- |    Source     |
-                      +---------------+ contains  +---------------+
-                              |
-                              |
-                      +----------------+
-                      | Transformation |
-                      +----------------+
+                                  +-----------------+
+                                  |     Project     |
+                                  +-----------------+
+                                          |  
+                                          |
+                                          | 
+                            +------------------------------+    
+                            |                              |      
+                            | 1:n                          | 1:n         
+                  +------------------+           +------------------+
+      +---------- |   FeatureName    |           |     Anchor       |
+      |      1:n  +------------------+           +------------------+
+      |                 ｜                             ｜          |
+      |                 ｜ 1:n                         ｜1:n       |
+      |                 +------------------------------+          |
+      |                               ｜                           |
+      |                               ｜1:n                       \|/ has
+      |   +--------------+    +---------------+                 +----------------+
+      |   |Transformation|----|    Feature    | --------------- |   DataSource   |            
```

Comment:
I think we need to remove the line from Feature -> DataSource/Source? It should be covered in the AnchoredFeature -> DataSource connection

Reply:
Removed those.